### PR TITLE
merge the penguins branch with the master branch

### PR DESCRIPTION
**Pull request by celinetanguy**

### CODE REVIEW

File: Penguins_chapter.qmd [hunk 1]
```diff
@@ -0,0 +1,1007 @@
...
+    toc: true 
+    toc_float: true 
+    toc_depth: 3
+---
+
+# Introduction {style="color:navy"}
+
+## Study context {style="color:royalblue"}
+
+This document is a report on the analysis of a dataset on penguins. The data collected contains informations on multiples [variables](#colum-name). The dataset was find on Kaggle : https://www.kaggle.com/datasets/utkarshx27/penguin-size-clutch-and-blood-isotope-data?select=penguins_raw.csv. This dataset was made from differents articles according by Gorman *et al.* [@10.1371/journal.pone.0090081] and Horst *et al.* [@10.32614/RJ-2022-020]. It focused on three penguin species : Adelie (*Pygoscelis adeliae*), Chinstrap (*Pygoscelis antarcticus*) and Gentoo (*Pygoscelis papua*). The dataset contains 344 observations and 17 variables. The goal of this analysis is to explore the dataset and to identify the main characteristics of the penguins.
```
Comment (ChloeVDS) [inline Penguins_chapter.qmd:19]: Little conjugation error : "was found" instead of "was find"

**Review by ChloeVDS — APPROVED** [review]
I read your work. It's well witten, the statistics and the results parts are brief and clear, I like them. I left a comment at the begginning of your work. I didn't see any other conjugation/spelling error and I don't have any other suggestions.
Well done ! I learnt some new things about penguins !

**Comment by MarieEtienne** [comment]
Don't wait longer for review, it's time to merge

File: Penguins_chapter.qmd (hunk 1 above)
Comment (ManonDenis) [reply Penguins_chapter.qmd:19]: Noted, we're going to change that

**Pull request merged**